prometheus-exporters/aci-suite/charts/data-collector/values.yaml

```diff
@@ -0,0 +1,48 @@
+fullname: aci-data-collector
+enabled: true
+
+replicaCount: 1
+
+image:
+  name: aci-data-collector
+  tag: "1.1.0"
+  pullPolicy: Always
+
+resources:
+  limits: {}
+  requests: {}
+
+nodeSelector: {}
+tolerations: []
+affinity: {}
```

Comment:
This is usually not needed, we don't do any node pinnings. Unless you have anti affinity rules in place, I'd say we scrap this.


```suggestion
```